test/fixtures/generator/bwb.xcodeproj/xcshareddata/xcschemes/generator.xcscheme

```diff
@@ -78,9 +78,9 @@
                <EnvironmentBuildable>
                   <BuildableReference
                      BuildableIdentifier = "primary"
-                     BlueprintIdentifier = "979D12680661F4B864602CE6"
-                     BuildableName = "tests.xctest"
-                     BlueprintName = "tests"
+                     BlueprintIdentifier = "85C876DB90D51CD225023BB2"
+                     BuildableName = "generator"
+                     BlueprintName = "generator"
```

Comment:
Seems this is non-deterministic. We should have it pick from a sorted list (in another PR to land before this change).

Reply:
https://github.com/buildbuddy-io/rules_xcodeproj/pull/1144